### DISCUSSION TITLE
Transition from 1847 to 1848

### DISCUSSION THREAD

**Discussion by psknick** [Stellarium/stellarium#814] [Q&A] [answered]
### Expected Behaviour

There should be no change in the image of the constellations during the transition from 1847 to 1848

### Actual Behaviour

When switching from 1847 to 1848, the pattern of the constellations changes to 30 days

### System
* Stellarium version: Stellarium-0.19.2-x86_64.AppImage
* Operating system: Linux Mint 19.2
* Graphics Card: No Graphics Card, processor Intel® Pentium® G860 

![2019-10-23-21:57:31](https://user-images.githubusercontent.com/44753018/67425297-68214100-f5e0-11e9-81fa-07148614ba68.jpg)
![2019-10-23-21:56:54](https://user-images.githubusercontent.com/44753018/67425299-68214100-f5e0-11e9-82b4-7296ff1e86fa.jpg)

[log.txt](https://github.com/Stellarium/stellarium/files/3763996/log.txt)

**Comment by gzotti** [comment]
Please triple check your time settings. 1848 is the first year where a notion of time zones (something else than local mean solar time) could become valid.

**Comment by psknick** [comment]
![2019-10-23-22:39:58](https://user-images.githubusercontent.com/44753018/67428230-68243f80-f5e6-11e9-8e29-7ed3e988e08d.jpg)
![2019-10-23-22:40:38](https://user-images.githubusercontent.com/44753018/67428247-6e1a2080-f5e6-11e9-99c3-63632601bf84.jpg)

**Comment by psknick** [comment]
![2019-10-23-22:44:49](https://user-images.githubusercontent.com/44753018/67428495-e41e8780-f5e6-11e9-90c0-6abecc7f2f54.jpg)

![2019-10-23-22:45:43](https://user-images.githubusercontent.com/44753018/67428504-ea146880-f5e6-11e9-8913-2b8ef6dad737.jpg)

**Comment by gzotti** [comment, marked as answer]
OK, so you must set a particular timezone. I wonder why your default timezone is UTC if your location is in Egypt? 

On my PC, the displayed time is LMST in 1847 and default zone time for 1848. If you are simulating the time for another location, you should set a fitting timezone or, for historical applications, prefer to use LMST. 

The timestamp in the lower button bar tells you the time zone in place.

**Comment by psknick** [comment]
By default, I use the user time zone UTC, because lunar and solar eclipses are given in reference books on the Internet for different dates and coordinates in UTC time. For example, a lunar eclipse on August 2, 1403 for Dender is given at UTC 22:37:05.

On the topic I understood - I got confused in time ... Thank you!
